mkosi/resources/man/mkosi.1.md

```diff
@@ -116,26 +119,31 @@ The following command line verbs are known:
 `journalctl`
 :   Uses **journalctl** to inspect the journal inside the image.
     Any arguments specified after the **journalctl** verb are appended to the
-    **journalctl** invocation.
+    **journalctl** invocation. To pass extra options, separate
+    them from regular options with `--`.
```

Comment:
```
All arguments specified after the `journalctl` verb and separated by `--` from the regular options are
appended to the **journalctl** invcation
```
This also changes the formatting of the first journalctl, since that is our verb and not the command.